src/guides/v2.3/ui_comp_guide/components/ui-masonry.md

```diff
@@ -45,8 +45,8 @@ The Masonry component has dependencies on the following components:
 
 Extends `Listing`:
 
-*  [app\code\Magento\Ui\view\base\web\js\grid\masonry.js]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/js/grid/masonry.js)
-*  [app\code\Magento\Ui\view\base\web\templates\grid\masonry.html]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/templates/grid/masonry.html)
+*  [app\code\Magento\Ui\view\base\web\js\grid\masonry.js](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Ui/view/base/web/js/grid/masonry.js)
```

Comment:
`master` on devdocs should not contain any links to unreleased magento2 codebase.